scripts/vcf_site_level_filtering_wdl/JointVcfFiltering.wdl

```diff
@@ -27,6 +27,8 @@ workflow JointVcfFiltering {
 		String indel_annotations
 		File? gatk_override
 
+		Boolean use_allele_specific_annotations = false
```

Comment:
I gave this a default, but I can also see an argument for having it be a required input since it's somewhat tied to the input annotations (which are currently required). Would love a second opinion.

Reply:
Ah makes sense, it does seem cleaner to keep the defaults in the command line.